client/src/components/login-form.js

```diff
@@ -1,106 +1,131 @@
-import React, { Component } from 'react'
-import { Redirect } from 'react-router-dom'
-import axios from 'axios'
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import axios from "axios";
+import Button from "react-bootstrap/Button";
```

Comment:
i feel like this is the first bootstrap component i see? why arent we using bootstrap more if 1 person is deciding to use the Button here? Shouldnt have this been said and people learn how to use react-bootstrap for all of the buttons to have a similar styling already provided?